### ISSUE TITLE
default DetailPopup does not disappear

### ISSUE THREAD

**Issue by dimabin** [nhn/tui.calendar#1336]
## Version

2.1.3
## Test Environment

Chrome
## Current Behavior

In my project, the useDetailPopup option is set true and the useFormPopup is false.
After selecting en event I get the default DetailPopup.
But if I click on "Edit" in this popup, the popup does not close.

```js
// Write example code
```

## Expected Behavior

Can I close the default DetailPopup somewhere in beforeUpdateEvent? If so, how?

**Comment by adhrinae** [comment]
@dimabin 
You're right. There's no public API to close popups at this moment. But you can access the store of the calendar directly. [(Reference)](https://github.com/nhn/tui.calendar/issues/555#issuecomment-1234928012)

And there is a `hideDetailPopup` action or you can leverage [any other actions](https://github.com/nhn/tui.calendar/blob/main/apps/calendar/src/slices/popup.ts#L26-L29).

```ts
cal.on('beforeUpdateEvent', () => {
  cal.getStoreDispatchers().popup.hideDetailPopup();
});
```

Similar to #555.
